### PR TITLE
UI Polish with Mike

### PR DESCRIPTION
**Pull request by caleybrock**
screen shot 2016-09-08 at 1 37 38 pm
- Change colours on a couple of icons
- Fix one inline style (TODO: make this a class in upstream)
- Rearrange order of a couple of icons
- Put layers controls and tooltip references behind a flag (TODO: push this change to upstream)

### CODE REVIEW

File: src/js/tools/drawing/ColorSwap.js [hunk 1]
```diff
@@ -14,6 +14,10 @@
       {key : 'ctrl', description : 'Apply to all layers'},
       {key : 'shift', description : 'Apply to all frames'}
     ];
+
+    if (!Constants.ENABLE_MULTIPLE_LAYERS) {
+      this.tooltipDescriptors.splice(0, 1);
+    }
```
Comment (islemaster) [inline src/js/tools/drawing/ColorSwap.js:16]: What about

``` javascript
this.tooltipDescriptors = [
  Constants.ENABLE_MULTIPLE_LAYERS && {key : 'ctrl', description : 'Apply to all layers'},
  {key : 'shift', description : 'Apply to all frames'}
].filter(function (x) { return x; });
```

Comment (islemaster) [inline src/js/tools/drawing/ColorSwap.js:16]:
``` javascript
this.tooltipDescriptors = [];
if (Constants.ENABLE_MULTIPLE_LAYERS) {
  this.tooltipDescriptors.push({key : 'ctrl', description : 'Apply to all layers'});
}
this.tooltipDescriptors.push({key : 'shift', description : 'Apply to all frames'});
```

**Comment by islemaster** [comment]
I prefer building up the tooltip contents instead of conditionally removing some.  Otherwise LGTM!

**Comment by caleybrock** [comment]
Updated to building up the tooltip and hid one more UI element (same way I did with hiding the layers list, I hid cloning layers under transforms). Going to merge.

**Comment by islemaster** [comment]
:+1:

**Pull request merged**